MO/MiddleOntology.rdf

```diff
@@ -78,8 +78,7 @@
 	
 	<owl:Class rdf:about="https://spec.edmcouncil.org/auto/ontology/MO/MiddleOntology/Location">
 		<rdfs:subClassOf rdf:resource="https://spec.edmcouncil.org/auto/ontology/MO/MiddleOntology/ImmaterialEntity"/>
-		<rdfs:label xml:lang="en">location</rdfs:label>
-		<rdfs:label xml:lang="en">place</rdfs:label>
+		<rdfs:label xml:lang="en">Location</rdfs:label>
```

Comment:
Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.